Fitting models:   0%|          | 0/11 [00:00<?, ?it/s]Fitting models:   9%|▉         | 1/11 [00:00<00:05,  1.78it/s]Fitting models:  18%|█▊        | 2/11 [00:04<00:21,  2.36s/it]Fitting models:  27%|██▋       | 3/11 [00:09<00:30,  3.80s/it]Fitting models:  36%|███▋      | 4/11 [00:09<00:16,  2.38s/it]Fitting models:  45%|████▌     | 5/11 [00:11<00:13,  2.25s/it]Fitting models:  55%|█████▍    | 6/11 [00:12<00:07,  1.58s/it]Fitting models:  64%|██████▎   | 7/11 [00:12<00:04,  1.16s/it]Fitting models:  73%|███████▎  | 8/11 [00:12<00:02,  1.16it/s]Fitting models:  82%|████████▏ | 9/11 [00:22<00:07,  3.69s/it]Fitting models:  91%|█████████ | 10/11 [00:22<00:02,  2.60s/it]Fitting models: 100%|██████████| 11/11 [00:25<00:00,  2.64s/it]Fitting models: 100%|██████████| 11/11 [00:25<00:00,  2.32s/it]
Done:  J1222-0007
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:325: RuntimeWarning: invalid value encountered in scalar divide
  ea = abs(errors[2] / self.grad)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:326: RuntimeWarning: invalid value encountered in scalar divide
  eb = abs(errors[1] * (1.0 - eps) / self.grad)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:329: RuntimeWarning: invalid value encountered in scalar divide
  self.ellip_err = (abs(2.0 * errors[4] * (1.0 - eps) / self.sma
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:332: RuntimeWarning: invalid value encountered in scalar divide
  self.pa_err = (abs(2.0 * errors[3] * (1.0 - eps) / self.sma
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:276: RuntimeWarning: divide by zero encountered in scalar divide
  a = up_coeffs[1] / self.sma / abs(sample.gradient)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:277: RuntimeWarning: divide by zero encountered in scalar divide
  b = up_coeffs[2] / self.sma / abs(sample.gradient)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:277: RuntimeWarning: invalid value encountered in scalar divide
  b = up_coeffs[2] / self.sma / abs(sample.gradient)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:296: RuntimeWarning: invalid value encountered in scalar divide
  b_err = abs(b) * np.sqrt((ce[2] / up_coeffs[2])**2 + gre**2)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:276: RuntimeWarning: invalid value encountered in scalar divide
  a = up_coeffs[1] / self.sma / abs(sample.gradient)
/home/insepien/miniconda3/envs/usc/lib/python3.11/site-packages/photutils/isophote/isophote.py:295: RuntimeWarning: invalid value encountered in scalar divide
  a_err = abs(a) * np.sqrt((ce[1] / up_coeffs[1])**2 + gre**2)
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
ModelObjectWrapper: about to call _model.CreateModelImage()...
Done: J1222-0007
/home/insepien/dualAGN/fit/plot_fit_with1d.py:37: UserWarning: All values for SymLogScale are below linthresh, making it effectively linear. You likely should lower the value of linthresh. 
  [fig.colorbar(im[i], ax=ax[i], shrink=0.7).ax.yaxis.set_major_formatter(FormatStrFormatter('%.2f')) for i in range(len(ax))]
/home/insepien/dualAGN/fit/plot_fit_with1d.py:73: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`). Consider using `matplotlib.pyplot.close()`.
  fig = plt.figure(figsize=(14, 4))
Done:  J1222-0007
